packages/prejournal/README.md

```diff
@@ -1,6 +1,29 @@
-⚠️ This package is an untested sketch for [prejournal](https://prejournal.org/) timesheets  sync integration.
+# timeld - Prejournal integration
```

Comment:
In view of earlier conversation, would suggest "PreJournal Connector for timeld".